### ISSUE TITLE
Tag list on small screen

### ISSUE THREAD

**Issue by Rurik19** [wallabag/wallabag#2559]
### Issue details

Taglist in matherial theme on small screens looks bad:

![image](https://cloud.githubusercontent.com/assets/1955806/20095115/f487ce02-a5c5-11e6-8f46-c27f92ea34e6.png)

When I set `min-width: 120px` to `<li>` it looks better

![image](https://cloud.githubusercontent.com/assets/1955806/20095393/21519390-a5c7-11e6-9cf9-6dc87c998e78.png)

### Environment

* wallabag version 2.1.3

### Steps to reproduce/test case

See it from smartphone

**Comment by nicosomb** [comment]
Oh so nice the first screenshot 😁 ! 

You can open a new PR ;-)

**Comment by Rurik19** [comment]
I understand now! There is funny bug in materializecss documentation. on (http://materializecss.com/grid.html) it sais 

> Note: For now, just know that the s1 stands for small-1 which in plain English means "1 column on small screens".

Really the matter is upside-down it is **s12** for one column in small screen :)

`.row .col.s12 {
  width: 100%;
  margin-left: auto;
  left: auto;
  right: auto;
}`

So we have to use **s6**, I think

**Comment by Rurik19** [comment]
No. **s6** makes one column. **s5** is fine:
![image](https://cloud.githubusercontent.com/assets/1955806/20097997/4bb6926e-a5d3-11e6-86dd-17043f55db46.png)

**Comment by j0k3r** [comment]
Go for the PR :+1:
